app/controllers/results_submission_controller.rb

```diff
@@ -44,6 +44,11 @@ def newcomer_dob_check
   end
 
   def compute_potential_duplicates
+    last_job_run = DuplicateCheckerJobRun.find_by(competition_id: params.require(:competition_id))
```

Comment:
How can you make sure that this `find_by` always delivers the _last_ job run, when there can be multiple job runs? The job checker run has a default scope to apply an `ORDER BY` clause, but I'm not sure whether `find_by` yields the first or the last entry in that ordered set.

Reply:
I've tested it, this is giving last entry. Also I'm using this same in the above method `last_duplicate_checker_job_run`.